src/wallets/router/auth/auth.routes.ts

```diff
@@ -0,0 +1,73 @@
+import { AllowanceAuthRequestView } from "~routes/auth/allowance";
+import { BatchSignDataItemAuthRequestView } from "~routes/auth/batchSignDataItem";
+import { ConnectAuthRequestView } from "~routes/auth/connect";
+import { SignAuthRequestView } from "~routes/auth/sign";
+import { SignatureAuthRequestView } from "~routes/auth/signature";
+import { SignDataItemAuthRequestView } from "~routes/auth/signDataItem";
+import { SignKeystoneAuthRequestView } from "~routes/auth/signKeystone";
+import { SubscriptionAuthRequestView } from "~routes/auth/subscription";
+import { TokenAuthRequestView } from "~routes/auth/token";
+import type { RouteConfig } from "~wallets/router/router.types";
+
+export type AuthRoutePath =
```

Comment:
Routes (across all apps) are now type-safe, which includes calls to `navigate()`, `<Link>` and `<Redirect>`.